dstep/translator/Context.d

```diff
@@ -33,8 +34,12 @@ class Context
     private TypedefIndex typedefIndex_ = null;
     private Translator translator_ = null;
     private Output globalScope_ = null;
+    private Cursor[string] typeNames_;
 
-    public this(TranslationUnit translUnit, Options options, Translator translator)
+    public MacroDefinition[string] macroDefinitions;
+    string macroLinkage = "extern (D)";
```

Comment:
Yes that was the aim, I kinda forgot to add a flag to the driver code. What do you think about that idea?